resources/services/security/settings.go

```diff
@@ -71,7 +71,7 @@ func fetchSecuritySettings(ctx context.Context, meta schema.ClientMeta, parent *
 				res <- v
 			} else if v, ok := item.AsDataExportSettings(); ok {
 				res <- v
-			} else if v, ok := item.AsAlertSyncSettings(); ok {
+			} else if v, ok := item.AsAlertSyncSettings(); ok { // nolint:revive (in this case revive has a false positive on early-return)
```

Comment:
`revive` wants to do `!ok return` inside this `else if` which is not that nice either